specifyweb/frontend/js_src/lib/components/Merging/Status.tsx

```diff
@@ -65,6 +65,11 @@ export function Status({
 
   const loading = React.useContext(LoadingContext);
 
+  if (state.status === 'SUCCESS') {
+    handleClose();
+    globalThis.location.reload();
+  }
+
```

Comment:
There already is a way for query builder to know that merging has completed:

https://github.com/specify/specify7/blob/8995d00014b85cbb8feccd0b8378208b910b3512/specifyweb/frontend/js_src/lib/components/Merging/index.tsx#L68-L73

It is supposed to re-run the query after merge. If it doesn't, it's a bug - please look into it

Also, you would notice that that code is rendered under the query builder URL, not merging URL, thus it has access to a callback for re-running the query and other goodies

Reply:
Also, re-running the query is far better than reloading the page:
- What if the query has unsaved changes?
- Plus, re-running the query takes far less time than reloading the page and then re-running the query